src/rust-2024/rustfmt-sorting.md

```diff
@@ -0,0 +1,43 @@
+# Rustfmt: Raw identifier sorting
+
+🚧 The 2024 Edition has not yet been released and hence this section is still "under construction".
+
+More information may be found in the tracking issues at <https://github.com/rust-lang/rust/issues/123800> and <https://github.com/rust-lang/rust/issues/123802>.
+
+## Summary
+
+`rustfmt` utilizes a new sorting algorithm.
+
+## Details
+
+The [Rust Style Guide] includes [rules for sorting][sorting] that `rustfmt` applies in various contexts, such as on imports.
+
+Previous versions of the Style Guide and Rustfmt generally used an "ASCIIbetical" based approach. In the 2024 Edition this is changed to use a version-sort like algorithm that compares Unicode characters lexicographically and provides better results in ASCII digit comparisons.
+
+For example with a given (unsorted) input:
+
+```rust,ignore
+use std::num::{NonZeroU32, NonZeroU16, NonZeroU8, NonZeroU64};
+use std::io::{Write, Read, stdout, self};
+```
+
+In the prior Editions, `rustfmt` would have produced:
+
+```rust,ignore
+use std::io::{self, stdout, Read, Write};
+use std::num::{NonZeroU16, NonZeroU32, NonZeroU64, NonZeroU8};
+```
+
+In the 2024 Edition, `rustfmt` now produces:
+
+```rust,ignore
+use std::io::{self, Read, Write, stdout};
+use std::num::{NonZeroU8, NonZeroU16, NonZeroU32, NonZeroU64};
+```
+
+[Rust Style Guide]: ../../style-guide/index.html
+[sorting]: ../../style-guide/index.html#sorting
+
+## Migration
+
+The change can be applied automatically by running `cargo fmt` or `rustfmt` with the 2024 Edition.
```

Comment:
If permissible, I'd like to avoid having to repeat the "here's how to run rustfmt" text in however many style/rustfmt pages we end up having.

Would it suffice to have this one-liner under our migration sections, and then just link out to the detailed instructions for how to do so, which will be added as part of #331?